Automattic/wp-calypso
# Add onboarding accessibility

#### Changes proposed in this Pull Request

- Added table headers
- Added table summary
- Define aria-labels for on input fields in onboarding flow
- Updated search icon colors for consistency

#### Testing instructions



1. Go to http://calypso.localhost:3000/start/site-type
2. Verify functionality is as expected

Fixes #34604
